docs/packages/only_wiki/android_log.rst

```diff
@@ -0,0 +1,16 @@
+.. spelling::
+
+    android_log
+
+.. _pkg.android_log:
+
+android_log
+===========
+
+File only available in wiki.
```

Comment:
It's a helper package, there will be no corresponding `hunter_add_package`.

List (created by `grep -IR hunter_find_helper cmake/find`):
* accelerate
* android
* android_log
* appkit
* assetslibrary
* audiotoolbox
* avfoundation
* coredata
* corefoundation
* coregraphics
* corelocation
* coremedia
* corevideo
* egl
* foundation
* glapi
* gles2
* gles3
* glkit
* imageio
* iokit
* metal
* mobilecoreservices
* opengles
* osmesa
* quartzcore
* uikit

Reply:
what shall we do with helpers? Should we add them in their own folder, because they are no hunter package?

Reply:
> Should we add them in their own folder

No, no need. The fact that they are special reflected by absense of `hunter_add_package` in example. We can add `.. note` section though.

Reply:
How about this note?
```
.. note::

    This is a helper package. There is no corresponding package in hunter to be included by ``hunter_add_package(...)``
```